Docs/source/install/hpc/s3df.rst

```diff
@@ -0,0 +1,189 @@
+
+  .. code-block:: bash
+
+    srun --pty --cpus-per-task=8 --ntasks-per-node=4 --gpus=4 --mem=100GB --nodes=1 --time=2:00:00 --partition=ampere --account=atlas bash
+
+
+
+
+2) Create & Populate the Conda Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+  .. code-block:: bash
+
+    # A) create env (example set; versions can vary by site repos)
+
+    conda create -n warpx-gpu-test -y python=3.11
+    conda activate warpx-gpu-test
+
+    #Get the full CUDA toolkit in your conda env.
+    conda install -c nvidia -y "cuda=12.9.*"
+
+    # Additionally install NVTX dev tools (cuda separates from the runtime version)
+    conda install -c nvidia -y "cuda-nvtx-dev=12.9.*"
+
+    # B) Install build tools
+    conda install -c conda-forge -y \
+      "cmake>=3.27" ninja make pkg-config git \
+      openmpi ucx mpi4py \
+      gcc_linux-64=13 gxx_linux-64=13
+
+    # C) Install ADIOS2 with OpenMPI (so we can use openpmd_backend = bp)
+    conda install -c conda-forge "adios2=*=mpi_openmpi*"
+
+    # D) Verify build tools
+    which cmake && cmake --version     # should show cmake >= 3.24 from your env
+    which nvcc  && nvcc  --version     # should show CUDA 12.9 from the nvidia 'cuda' metapackage
+    which mpicxx && ompi_info | head   # Open MPI from conda-forge
+
+
+
+.. note::
+  - ``cuda-nvtx-dev`` provides the nvtx3 headers used by recent NVTX versions.
+  - ``cuda-nvcc`` brings nvcc under ``$CONDA_PREFIX/bin/nvcc``.
+  - MPI here is via ``mpich`` in the env (``mpicc``, ``mpicxx`` will be at ``$CONDA_PREFIX/bin/...``).
+
+
+3) Download WarpX
+^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+  git clone https://github.com/BLAST-WarpX/warpx.git
+  cd warpx
+
+
+
+
+4) Environment Fixes (NVTX headers + dlopen/dlsym)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Recent NVTX installs the header under ``nvtx3/nvToolsExt.h``. Add it to your include path, and ensure ``libdl`` is linked (``CUDA/NVTX`` uses ``dlopen()/dlsym()`` dynamically):
+
+.. code-block:: bash
+
+  # ensure libdl is not dropped by the linker (GCC + gold/ld with --as-needed)
+  export LDFLAGS="${LDFLAGS} -Wl,--no-as-needed -ldl"
+
+  # verify the header exists (one of these should print a path)
+  ls "$CONDA_PREFIX/targets/x86_64-linux/include/nvtx3/nvToolsExt.h" \
+   || ls "$CONDA_PREFIX/include/nvtx3/nvToolsExt.h"
+
+  # include & library paths from the conda CUDA toolchain
+  export CUDACXX="$CONDA_PREFIX/bin/nvcc"
+  export CUDA_HOME="$CONDA_PREFIX"
+
+  export CPATH="$CONDA_PREFIX/targets/x86_64-linux/include/nvtx3:$CONDA_PREFIX/targets/x86_64-linux/include"
+  export LIBRARY_PATH="$CONDA_PREFIX/targets/x86_64-linux/lib"
+  export LD_LIBRARY_PATH="$CONDA_PREFIX/targets/x86_64-linux/lib"
+
+  # help CMake find ADIOS2/openPMD (if using conda-provided openPMD)
+  export ADIOS2_DIR="$CONDA_PREFIX"
+  export openPMD_DIR="$CONDA_PREFIX"
```

Comment:
I usually find that `conda activate` sets the `CMAKE_PREFIX_PATH` environment variable, which CMake uses to find all dependencies. Thus, I think the quirks mentioned in section 4) might actually not be needed. Can you try again?